manim/mobject/geometry.py

```diff
@@ -938,15 +988,98 @@ def __init__(self, start_anchor, start_handle, end_handle, end_anchor, **kwargs)
 
 
 class Polygon(VMobject):
+    """A shape created by defining its vertices.
+
+    Parameters
+    ----------
+    vertices : :class:`list`
+        The vertices of the mobject. The first one is repeated to close the shape. Must define 3-dimensions: `[x,y,z]`
+    color : :class:`str`, optional
```

Comment:
```suggestion
    color : :class:`~.Colors`, optional
```

Reply:
I've used `` :class:`str` `` quite a few times in the previous PR to represent color, should i go back and change those instances to ``:class:`~.Colors` `` too?

Reply:
Sorry I'm just now seeing this. Must've missed the notification. Yes, the previous instances (if they haven't already been merged) should be updated in your other PRs. Otherwise, a new smaller PR can be made.

Reply:
I assumed that the answer would be yes 😅 so I went back and changed them already.